ansible/deploy.yml

```diff
@@ -190,47 +181,36 @@
       when: dash_network != "mainnet" and dash_network != "testnet"
       delegate_to: '{{ play_hosts | first }}'
 
-# Start platform
+# Start masternodes
 
-- name: Set up platform on seed nodes
-  hosts: seed_nodes
-  become: true
-  pre_tasks:
-    - name: Check inventory for masternodes
-      ansible.builtin.set_fact:
-        node: "{{ seed_nodes[inventory_hostname] }}"
-      tags: always
-      when: inventory_hostname in seed_nodes
-    - name: Check inventory for seed nodes
-      ansible.builtin.fail:
-        msg: Seed node not defined in network config
-      when: node is not defined
-  roles:
-    - role: mn_evo_services
-      tags:
-        - evo
-      when: evo_services
-    - role: elastic_beats
-
-- name: Set up platform on masternodes
+- name: Set up core and platform on masternodes
   hosts: masternodes
   become: true
   pre_tasks:
     - name: Check inventory for masternodes
       ansible.builtin.set_fact:
+        masternode: "{{ masternodes[inventory_hostname] }}"
         node: "{{ masternodes[inventory_hostname] }}"
       tags: always
       when: inventory_hostname in masternodes
     - name: Fail if no masternodes present
       ansible.builtin.fail:
         msg: Masternode not defined in network config
-      when: node is not defined
+      when: masternode is not defined
```

Comment:
Where masternode OR node?

Reply:
There's no real way to reach this logic branch, all masternodes and seed nodes are always defined on all networks. I find the names for these checks a bit weird as well, if we ever need to make a network without seed nodes or masternodes we will just comment out the tasks deploying roles to the relevant nodes instead. It's a lot of error boilerplate for no real benefit; I'll do a commit to show what I think can be removed, then I think we can merge this :+1: